### ISSUE TITLE
Clauses fail to load via API

### ISSUE THREAD

**Issue by jierdin** [openlawteam/openlaw-issue-tracker#26]
TL;DR: There is some issue with loading any Clauses on etherizeit instance.  Note, that this Hybrid Formation Service Agreement-bug  seems to run fine on OpenLaw's interface, but fails to load any component Clause when called from the API.

I have been loading a 'Formation Service Agreement' from the etherizeit OL instance successfully.
I tried to make a 'Hybrid Formation Service Agreement' with two fragments, including the original agreement. However, the browser console throws an error, saying that the clause template could not be loaded (and, unsurprisingly, nothing appears). 
_":3000/_next/static/runtime/main.js?ts=1579396557641:10034 Openlaw Execution Error: the template formation service agreement was not loaded"_

I was wondering if it might be an issue of capitalization mismatch, so I tried  changing the template to only include the existent 'service-agreement-formation' template, but that would not load either. 

This issue of Clauses not loading might be related to my earlier issue, which prevented us from being able to load a deal. It also might be related to the way in which OL seems to point back to the original instance even though there is not always an agreement there... When I hit 'refresh' on my browser when looking at any of etherizeit templates, say, https://lib.openlaw.io/web/etherizeit/template/Hybrid%20Formation%20Service%20Agreement, it will then point me to 
https://lib.openlaw.io/template/Hybrid%20Formation%20Service%20Agreement

Replicating this bug on your end is unfortunately  not entirely trivial, since we have some local variables that aren't shared on the repo. If you can't see the same issue from using Clauses (or Deals!) on another private instance, let me know and I will publish a link on Etherize's live site for you to be able to  bang into this issue with the  "Hybrid Formation Service Agreement-bug" template. 

(I will probably be modifying the Hybrid Formation Service Agreement to not use fragments meanwhile)
